### DISCUSSION TITLE
TODO里所提到的图片广场在哪开启呢

### DISCUSSION THREAD

**Discussion by Matinal111** [lsky-org/lsky-pro#418] [问答]
如题，后台每个选项都看了，没找到这项功能

**Comment by penndu** [comment]
![image](https://user-images.githubusercontent.com/46226528/167575575-bed983e2-a46f-42e7-a0b2-c621e6262471.png)

**Reply by Matinal111** [reply]
是这个啊，我还以为图片广场是一个可以浏览所有图片的公开链接

**Reply by penndu** [reply]
> 是这个啊，我还以为图片广场是一个可以浏览所有图片的公开链接

需要用户自行公开才行
